src/theme.ts

```diff
@@ -191,48 +200,48 @@ export const theme: Theme = {
       // original Figma fontSize: 40px
       fontSize: pxToRem(32),
       [sm]: {fontSize: pxToRem(32)},
```

Comment:
xs/sm teda ako su, dokonca vacsie ako md? ci to je, ze uz toho bolo moc a je to na follow-up?

Reply:
no nezabudol, H1 je pouzite len pre nadpis v topbare, kde to vo Figme pre mobil je naozaj take obrovske

akoze mozeme ignorovat ten mobilny design "uplne", nvm co myslis?

Reply:
pre porovnanie
![image](https://github.com/ZdruzenieSTROM/webstrom-frontend/assets/5648837/d466bd65-7032-476d-8063-d78c43e59b1a)
![Uploading image.png…]()